parkhangs-webapp/package.json

```diff
@@ -10,6 +10,7 @@
     "lodash": "^4.17.19",
     "moment": "^2.27.0",
     "react": "^16.13.1",
+    "react-add-to-calendar": "^0.1.5",
```

Comment:
Looks like this page just sends the information to the Google Calendar and doesn't actually automatically create the event with the button click. Is this what we want?

Reply:
I tried implementing it from scratch with the google calendar api at first and a lot of it tied into oath at runtime, where the gapi no longer applies outside of the componentDidMount. Putting the entire functionality in onclick had the same issue where gapi is not defined.

If you have another package or source in mind, we can switch to that. But I think for this progress report this would suffice.

Reply:
I can look at alternatives too and keep cracking away at it after my appointment

Reply:
@cmaija @philchapdelaine what do you guys think?

Reply:
I think we can just use this for now and sort it out next week